packages/salesforce-adapter/src/transformers/transformer.ts

```diff
@@ -875,6 +875,12 @@ export class Types {
   }
 }
 
+export const isFormulaField = async (field: Field): Promise<boolean> => {
+  const formulaTypes = Object.keys(Types.formulaDataTypes)
+  const fieldType = (await field.getType()).elemID.typeName
+  return formulaTypes.includes(fieldType)
```

Comment:
Maybe something like this will be more efficient? 

```suggestion
  const fieldType = (await field.getType()).elemID.typeName
  return Types.formulaDataTypes.hasOwnProperty(fieldType)
```

Reply:
ESLint actually [warns](https://eslint.org/docs/latest/rules/no-prototype-builtins) about using `hasOwnProperty` directly and wants me to use the more cumbersome
`return Object.prototype.hasOwnProperty.call(Types.formulaDataTypes, fieldType)` or disable the lint warning explicitly.
Given that it's not clear which alternative is actually faster for the case where we have O(10) properties (See [this](https://stackoverflow.com/questions/30326452/why-is-object-keys-faster-than-hasownproperty), [this](https://dev.to/ethanarrowood/which-is-faster-objhasownpropertyprop-vs-objectkeysobjincludesprop-5704) and [this](https://www.measurethat.net/Benchmarks/Show/2294/0/hasownproperty-vs-objectkeys)) I'd rather leave the code as-is if it's all the same to you (unless I missed something?)